src/recap

```diff
@@ -252,7 +252,7 @@ plugins_info(){
 plugins_load() {
   local plugin_dir="$1"
   log INFO "Loading plugins from: ${plugin_dir}"
-  for plugin in "${plugin_dir}"/*; do
+  for plugin in $( ls -1 "${plugin_dir}" ); do
```

Comment:
This is what I used to do: use `for` loops for going over files, but if the files have spaces in the name, then that becomes an issue. My solution was to temporarily reset the `IFS` to a new line, but I didn't like that.

For most use cases I started doing this:
```
command 2>/dev/null | while read var; do
  echo "${var}";
done
```
This reads the entire line including spaces instead of splitting on spaces as the `for` loop does due to the IFS. The only issue I've had with this is not being able to use variables outside the loop that were declared inside it (due to the loop being in a subshell).

The way I do it now, which seems best is:
```
while read var; do
  echo "${var}"; 
done < <(command 2>/dev/null)
```
Why? The loop isn't in a subshell so I can use variables without a problem; and I either get data to work with or not (with no errors).

I'd change the `for` loop to a `while` loop for this reason.